packages/react-api/src/types.d.ts

```diff
@@ -43,6 +43,10 @@ export type SourceProps = {
   type: MapTypesType['QUERY'] | MapTypesType['TABLE'] | MapTypesType['TILESET'];
   connection: string;
   geoColumn?: string;
+  dataResoultion?: number;
```

Comment:
Seems like there's a typo and should be `dataResolution` instead of `dataResoultion`